packages/access-client/package.json

```diff
@@ -95,18 +99,32 @@
   },
   "eslintConfig": {
     "extends": [
-      "./node_modules/hd-scripts/eslint/index.js"
+      "eslint:recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:jsdoc/recommended"
     ],
     "parserOptions": {
       "project": "./tsconfig.json"
     },
     "rules": {
-      "unicorn/prefer-number-properties": "off",
-      "unicorn/prefer-export-from": "off",
-      "unicorn/expiring-todo-comments": "off",
-      "unicorn/no-array-reduce": "off",
-      "unicorn/explicit-length-check": "off",
+      "@typescript-eslint/no-unused-vars": "off",
+      "@typescript-eslint/ban-ts-comment": "off",
       "no-continue": "off",
+      "jsdoc/check-values": "off",
```

Comment:
I copied these rules over from hd-scripts so this PR doesn't have to make lots of changes to the src, but over time I'd prefer to just use the default "plugin:jsdoc/recommended" rules.